rules/integrations/google_workspace/persistence_google_workspace_user_organizational_unit_changed.toml

```diff
@@ -26,7 +26,51 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "Google Workspace User Organizational Unit Changed"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Google Workspace User Organizational Unit Changed
+
+An organizational unit is a group that an administrator can create in the Google Admin console to apply settings to a specific set of users for Google Workspace. By default, all users are placed in the top-level (parent) organizational unit. Child organizational units inherit the settings from the parent, but can be changed to fit the needs of the child organizational unit.
+
+Permissions and privileges for users are often inherited from the organizational unit they are placed in. Thus if a user is changed to a separate organizational unit, they will inherit all privileges and permissions. User accounts may have unexpected privileges when switching organizational units that would allow a threat actor to gain a stronger foothold within the organization. The principle of least privileged (PoLP) should be followed when users are switched to different groups in Google Workspace.
```

Comment:
```suggestion
Permissions and privileges for users are often inherited from the organizational unit they are placed in. Therefore, if a user is changed to a separate organizational unit, they will inherit all privileges and permissions. User accounts may have unexpected privileges when switching organizational units that would allow a threat actor to gain a stronger foothold within the organization. The principle of least privileged (PoLP) should be followed when users are switched to different groups in Google Workspace.
```